lib/Gedmo/Translatable/TranslatableListener.php

```diff
@@ -480,7 +505,7 @@ public function postLoad(EventArgs $args)
                     }
                 }
                 // update translation
-                if ($is_translated
+                if (($is_translated && (!empty($translated) || !$this->fallbackEmptyTranslations))
```

Comment:
`$translated` is supposed to be a string, which is the reason to use `empty()` on it?